### DISCUSSION TITLE
[Feature] streaming to chromecast

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#7030] [Feature Request]
### The feature

shared albums, adding assets from these albums to private collections, streaming assets or albums from Android via Chromecast to a TV - this is how we use Google Photos in our family. It would be great to do this with immich to allow to switch all family members from Google Photos to immich.

This is not a duplicate of #1614
I already asked to reopen #2575, but a new request seams to be easier to create.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by laxmanpradhan** [comment]
Yes please. An android tv app requires an app for each device. Many devices are already Chromecast enabled. This would allow sharing photos on anyone's tv when visiting other friends or family.

**Reply by tmein** [reply]
This is my use case too. I want to share pictures with family who own chromecasts. My immich is not accessible without VPN so the android TV app is just not a good solution, too many hoops to jump through and then undo just to put an image on a big screen.

**Reply by Klopperz** [reply]
+1

Although I've exposed immich through a ngnix proxy and slapped a letsencrypt cert to it.

Even if you make casting available for direct connections (server)<->(cast device) would enable all home use only cases already 🥳

**Comment by hugoduraes** [comment]
This would be a very nice addition to Immich on mobile!

**Comment by Se7enair** [comment]
Dont forget about the apple users ;)
Apple TV, FireTV, and all the other more or less well-known streaming/casting devices.

**Reply by CiNcH83** [reply]
> Dont forget about the apple users ;) Apple TV, FireTV, and all the other more or less well-known streaming/casting devices.

With Apple, Google and Amazon backing [Matter](https://en.wikipedia.org/wiki/Matter_(standard)), there is hope that [Matter Casting](https://matter-smarthome.de/en/know-how/what-is-matter-casting/) will become a thing one day in order to have one common standard for casting to a TV via mobile device.

**Comment by JMendyk** [comment]
Now that Immich is a part of FUTO, adding FUTO's FCast integration would also be great - integrating with an open standard-based casting solution.

**Comment by gromgsxr** [comment]
This would get my vote to it's about the only thing I found that I'm missing

**Comment by omricarmel** [comment]
https://github.com/immich-app/immich/pull/14497

This is a start. 
Works with chrome and casting publicly accessible assets (i.e. share without password)

**Comment by ChristianNewton** [comment]
Just wanted to add that this would be of great use to my family and I!

**Comment by lbouriez** [comment]
Would be great, it's one of the only features we are missing compared to Google photos.

**Comment by Vanuan** [comment]
This is a single feature that differentiates Google Photos from tons of other photo apps. A simple SMB server with photo viewing is all that is needed for a personal photo archive. All these AI search and album management is all just vendor lock in. We already have the date and EXIF metadata.

**Comment by cainenielsen** [comment]
I would love to be able to cast my photos and videos from Immich from my Android or web client to my Chromecast enabled TVs in my home.

**Comment by jrasm91** [comment]
Implemented with #18231 and #18341
